src/search/identity_search.rs

```diff
@@ -20,8 +20,8 @@ pub fn identity_search(
 
     let query = build_identity_query(query_descriptors, extra_query, scaffold_matches);
 
-    // Note: default to a large number of possible results to ensure we don't miss the molecule
-    let tantivy_limit = 10_000;
+    // Set arbitrarily large number to force an entire database search
+    let tantivy_limit = 1_000_000_000;
```

Comment:
This is a very big jump from 10k. Is there any concern we'd actually get close to 1mil?

Reply:
If we had 1mil would it blow up our memory too big and crash cheminee?

Reply:
Good point. Alright, maybe we can compromise with 100_000 to be comprehensive but also make sure we're safe memory-wise. The 100_000 would correspond to the length of a `Vec<(Score, DocAddress)`.